src/Component/Card-note/CardNote.jsx

```diff
@@ -0,0 +1,53 @@
+import { useNote } from "../../Context/Note/note-context";
+import { useState } from "react";
+import "../../Style/cardNote.css";
+import Typographycal from "../Typographcal/Typographycal";
+import Palette from "../Palette/Palette";
+import Tags from "../Tags/Tags";
+import Priority from "../Priority/Priority";
```

Comment:
Use Barrel exports here.

Reply:
yes I will use this method .
thank you.

Reply:
OkK forgot to do that.